stormpath/stormpath-sdk-react
# Fix SocialLoginLink redirection to authenticated home route

Fixes so that when the SocialLoginLink redirects back from the callback, it does so to the specified authenticated HomeRoute.
#### How to verify
1. Checkout this branch and run `$ npm run build-cjs`.
2. Download and use the [React SDK example application](https://github.com/stormpath/stormpath-express-react-example).
3. Link from the example application to the React SDK branch.
4. Run the example application (`$ npm start`) and open you browser and navigate to `http://localhost:3000/login`.
5. Login with one of the social login buttons.
6. Verify that you are redirected to `/profile` (the authenticated home route).

Fixes #96
